pytest_httpserver/pytest_plugin.py

```diff
@@ -1,23 +1,7 @@
-
-
 import os
 
 import pytest
-from .httpserver import HTTPServer
-
-
-class Plugin:
-    SERVER = None
-
-
-class PluginHTTPServer(HTTPServer):
-    def start(self):
-        super().start()
-        Plugin.SERVER = self
-
-    def stop(self):
-        super().stop()
-        Plugin.SERVER = None
+from pytest_httpserver import HTTPServer
```

Comment:
Could you re-write it to `from .httpserver import HTTPServer`? I would like to avoid circular imports if possible and here importing from the same-level module is better I think.